interfaces/openfga/v0/schema.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+"""This file defines the schemas for the provider and requirer sides of the openfga interface.
+
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"openfga":
+                 {"address":  "10.10.0.17",
+                  "port": "8080",
+                  "scheme": "http",
+                  "token": "test-token",
+                  "store_id": "01GK13VYZK62Q1T0X55Q2BHYD6",
+                 }
+             }
+
+    RequirerSchema:
+        unit: <empty>
+        app: {"store_name": "test-store-name"}
+"""
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel, Field
+
+
+class OpenFGAProviderData(BaseModel):
+    address: str = Field(
```

Comment:
This can be an `IPvAnyAddress` instead of just a string. It's still a string btw but with some kind of regex validation. There's an example of this in [this PR](https://github.com/canonical/charm-relation-interfaces/pull/80)

Reply:
done